### DISCUSSION TITLE
How was the documentation website build ?

### DISCUSSION THREAD

**Discussion by aahnik** [h2oai/wave#426] [Q&A] [answered]
I mean which stack or which static site generator ?

Which template ?

Or was it custom made by you guys ?

It looks really great.❤️

**Comment by lo5** [comment, marked as answer]
Thanks :-)

The docs were built using Docusaurus 2: https://v2.docusaurus.io/ 
The API docs use a custom markdown generator based on pdoc: https://pdoc3.github.io/pdoc/
